test/anyOf_test.js

```diff
@@ -300,6 +300,109 @@ describe("anyOf", () => {
     expect(node.querySelector("select").value).eql("1");
   });
 
+  it("should not change the selected option when entering values", () => {
```

Comment:
I think you should add a test just like this, but when the `anyOf` keyword already exists within one of the `anyOf` branches?

Reply:
Similarly a test in which the `allOf` keyword exists? (if that's even possible to do, given that rjsf doesn't support allOf yet)

Reply:
Good idea, I'll add them in